### ISSUE TITLE
Compact Machine Island unplayable without cheats + FTB Skies 2

### ISSUE THREAD

**Issue by LouisOuellet** [FTBTeam/FTB-Modpack-Issues#5567]
### Modpack

FTB Skies

### Modpack version

1.6.0

### Has the pack been modified

No

### Describe the bug

https://github.com/FTBTeam/FTB-Modpack-Issues/issues/5214 & https://github.com/FTBTeam/FTB-Modpack-Issues/issues/5512, still persist after latest update.

An Island Specific Quest has been removed. This quest used to provide the necessary items required for the compact machine island. Including the shrinking tool, dirt, an ice block and a lava bucket. Used to also come with a seed allowing grass to grow.

Please take this bug report seriously and stop dismissing it.

### Steps to reproduce

Create Compact Machine Island in 1.6.0

### Expected behaviour

An additional quest should complete upon spawning on the pillar

**Comment by jon419** [comment]
Same issue for me.  Last version it worked on is 1.4.1.  Looks like it broke in April with the 1.5.0 update.

In 1.4.1 you start on the compact machine instead of on a pedestal with the compact machine in the pedestal.

Also, in 1.4.1 you are immediately given a satchel with the following:

- Charm of Levitation
- 9 Infused Dirt
- 16 Flax Seeds
- 4 Ice
- 1 Easter Egg
- 32 Rice Panicle
- Iron Wand
- Water Condensator

In 1.5.0+ you just land on a pedestal and have to figure out how that you need to pick up the compact machine and place it down next to the pedestal.  And then you can't proceed because you don't have any items to progress with.

**Comment by MaienM** [comment]
The second quest in the "I - Getting Started" chapter (book icon, titled "Join or Create a team island") gives you a personal shrinking device and a few other items. In addition, the "Emergency Items" option on the far right of the quest screen can be used to get another personal shrinking device if you lose it.

This is not made super clear (I was confused for a bit as well yesterday), so perhaps having text/a sign/an additional quest to point you in this direction might be useful, but do I think everything you need to get started is available this way already.

**Comment by LouisOuellet** [comment]
Nope, you don't get anything that is required to start a sky world. A shrinking device only allows you to shrink. Doesn't give you dirt, water and lava

**Comment by MaienM** [comment]
You can sneak right-click the compact machine walls to get rocks and soil (which can be crafted into cobblestone, dirt, and gravel), just like you can on dirt. (This is actually called out in the 3rd quest that instructs you on how to do this in general.)

The quest which provides you with the shrinking device also provides you with a block of ice, which you can break to get water. Ice is also one of the options available in the emergency items section in case you lose this.

Lava is not something you start out with on any of the islands, it's something you'll work on getting as part of the "I - Getting Started" section of the quests.

**Comment by jon419** [comment]
The quest that gives you the shrinking device called "Starting Your Island" only gives you Bottle o' Experience, Annotated Codex, and the Personal Shrinking Device:

![image](https://github.com/user-attachments/assets/f2f41282-2b92-4128-b819-e3142dd671e7)

I don't see the emergency items quest/section:

![image](https://github.com/user-attachments/assets/ab17a081-b5e2-4bc1-9f6b-c39d874f0741)

The Compact Machine has no light like in previous versions and you cannot sneak right click the walls to get the pebbles for dirt and cobble.  There is also no quest that directs you to do this.

![image](https://github.com/user-attachments/assets/aa22174d-4e8b-4ba1-8557-3b7b7b3010b2)

**Comment by jon419** [comment]
For even more clarity of the issue - here is a video from the very start to getting into the machine and trying to sneak right click on all the walls.  For fun, i even did the sneak and left click....obviously you can't see that I am right clicking as there is no animation for that, but I promise, I am wildly clicking that  right mouse button on the walls and the floor.

https://www.youtube.com/watch?v=UfvX6WB7UUY

**Comment by jon419** [comment]
Just to be clear (even though the bug report already states this clearly) - this is for FTB Skies, NOT FTB Skies Expert.  FTP Skies Expert works exactly as @MaienM has noted.  FTB Skies Compact Sky Island does NOT work.

If I had to guess, FTB attempted to merge portions of FTB Skies Expert into FTB Skies, broke FTB Skies, and are ignoring the bug reports hoping to get it fixed.

**Comment by MaienM** [comment]
Ah, I see the issue... I misread your initial query. I'm playing FTB Skies Expert, and you're playing the non-expert version. I also faced difficulties starting with the compact machine island in that version, so when I saw this issue I immediately thought "oh I know this one, I just went through the same confusion", and I failed to notice the omission of the word "Expert" in the issue. My bad!

I just booted up the non-expert version and yeah, that all tracks. Some of these differences are quite surprising, like the omission of any kind of emergency tools. You generally wouldn't expect an expert version of the same pack to be more forgiving in some aspect such as that.

Anyway, none of that helps you. Sorry for the noise & confusion!

**Comment by jon419** [comment]
Has this one been seen by the FTB team?  Is there a way to join the FTP team as a contributor to fix issues and create PRs?

**Comment by LouisOuellet** [comment]
A new update has been released since. But the issue has yet to be resolved. Anyone?

**Comment by unnecessarymb** [comment]
Can't reproduce in Skies 2, emergency items are provided and the island starts with what you need.

**Comment by jon419** [comment]
This only happens in FTB Skies.  I am playing FTB Skies 2 with a compact machine with no issue.  I would love to see this fixed on FTB Skies though, still.

**Comment by ArtpokeBr** [comment]
Well, it turns out you indeed could not collect pebbles from Compact Machine's Wall Blocks, I've just fixed this.

Also, edited the quest description to hopefully help people a little bit.

<img width="775" height="946" alt="Image" src="https://github.com/user-attachments/assets/e10ebc55-9f49-480d-b953-7199f2dd40b7" />

**Comment by LouisOuellet** [comment]
Not fixed in FTB Skies 2. Right-Clicking the compact cube wall does not generate the pebbles.
